README.md

```diff
@@ -2,30 +2,68 @@
-3. **Centralized Prioritization**: At the start of each release cycle, all accepted VEPs will be designated as the project’s priority, focusing community efforts on the associated pull requests. Acceptance will be based on community support and a commitment to implementation.
-4. **Visibility and Tracking**: The Author of an accepted VEPs will open an issue to track their progress, maturity stages (alpha, beta, GA), list the associated bugs, and user feedback
-5. **Single source of truth**: Each VEP will be the authoritative reference for the associated feature. This aligns with the Kubernetes KEP process. It will ensure that each enhancement includes all the relevant information, including the design and the state.
+**Glossary**
+- VEP
+- EF
+- FF
+- RC
+- Alpha/Beta
+- release team
 
-The VEP owner is responsible to update it as its development progresses, until it is fully mature (or deprecated).
+## Process (DRAFT 2nd iteration)
 
-**NOTE**
-Acceptance of enhancement doesn't guarantee that the feature will land in the current or later release. The process is collaborative effort between contributor/s and Approvers. Features not landing to release branch prior to FF will be reverted based on their state (GA/Beta/Alpha, Off/On by default) that reflects how harmfull keeping the code is to the system.
+1. **VEP Creation**: VEP authors will initiate proposals via PRs to the `kubevirt/enhancements` repository. See what is required for every VEP - [Design proposal template](https://github.com/kubevirt/community/blob/main/design-proposals/proposal-template.md), each section needs to be filled even if the section doesn't apply to the VEP. 
 
-## Approvers responsibility
+2. **Visibility and Tracking**: The Author of VEP will open an issue to track their progress, maturity stages (alpha, beta, GA), list the associated bugs, and user feedback. 
+See what is required for every VEP's issue [Issue template](https://github.com/kubevirt/community/blob/main/design-proposals/proposal-template.md), each section should be filled.
 
-1. After Feature Freeze, approvers needs to go over issue and perform the following checklist
+3. **SIG Review and Collaboration**: Each VEP will have a target SIG or more in case of cross SIG feature, and the SIGs will assign a dedicated reviewer to oversee the proposal, collaborate with others SIGs as needed, and provide feedback or veto when necessary. The author and reviewers work towards merged VEP.
+
+Each SIG (Compute, Network, Storage) should sign-off the VEP with LGTM from representative (Sig-chair or approver of the sig) even if the proposal doesn't affect them. The sign-off works here as acknowledgment of the changes. 
+
+4. **Centralized Prioritization**: At the start of each release cycle, all accepted VEPs will be designated as the project’s priority, focusing community efforts on the associated pull requests. Acceptance will be based on community support and a commitment to implementation.
```

Comment:
I'm not sure this step is necessary, but if it is, I think it should be temporary, and we should state it would be removed when the process matures. IMO eventually (or, even now?) SIGs should have the sovereignty to prioritize VEPs on their own.